include/mbedtls/ssl_internal.h

```diff
@@ -1280,25 +1280,24 @@ int mbedtls_ssl_write_record( mbedtls_ssl_context *ssl, uint8_t force_flush );
 #if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL)
 int mbedtls_ssl_read_certificate_process(mbedtls_ssl_context* ssl);
 int mbedtls_ssl_write_certificate_process(mbedtls_ssl_context* ssl);
-#else
+#endif /* MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL */
```

Comment:
The pattern here should be different:
```c
#if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL)
tls 1.3 specific things
#elif defined(MBEDTLS_SSL_PROTO_TLS1_2_OR_EARLIER)
tls 1.2 specific things
#endif
common things
```

From what I understand `mbedtls_ssl_parse/write_certificate()` are specific to the TLS 1.2 code?